tests/containers/firewall.pm

```diff
@@ -51,7 +52,7 @@ sub run {
     # Stop the firewall if it was started by this test module
     if ($stop_firewall == 1) {
         systemctl('stop ' . $self->firewall());
-        systemctl('restart docker') if ($runtime eq "docker");
+        systemctl('restart docker') if ($runtime eq "docker" && !is_jeos);
```

Comment:
IMHO we cannot exclude an existing step (here for jeos), until affecting error is clarified, but unless there is a motivation to proceed anyway and we want to force test to continue.